### PR TITLE
Fi

### PR DESCRIPTION
**Pull request by an7oine**
This PR/Translation is about #151 .

**files translated**:
  - [x] `def/fi/about.cson`
  - [x] `def/fi/context.cson`
  - [x] `def/fi/menu_darwin.cson`
  - [x] `def/fi/menu_linux.cson`
  - [x] `def/fi/menu_win32.cson`
  - [x] `def/fi/settings.cson`
  - [x] `def/fi/welcome.cson`

**validation**
  - [x] I've validated my translation locally by running

`npm run validation -- --locale fi`

### CODE REVIEW

File: def/fi/menu_linux.cson [hunk 1]
```diff
@@ -0,0 +1,305 @@
+Menu:
+  "&File":
+    value: "Tiedosto"
```
Comment (liuderchi) [inline def/fi/menu_linux.cson]: Please preserve hotkey prompt (in this case `&F`) in the word or wrapped in braces

for example, [`value: "Datei(&F)"` (de/menu_linux.cson)](https://github.com/liuderchi/atom-i18n/blob/master/def/de/menu_linux.cson#L3)

Ref. [no3 in CONTRIBUTING.md](https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md#translation-guide)

File: def/fi/menu_win32.cson [hunk 2]
```diff
@@ -0,0 +1,307 @@
+Menu:
+  "&File":
+    value: "Tiedosto"
```
Comment (liuderchi) [inline def/fi/menu_win32.cson]: Please preserve hotkey prompt (in this case `&F`) in the word or wrapped in braces

for example, [`value: "Datei(&F)"` (de/menu_linux.cson)](https://github.com/liuderchi/atom-i18n/blob/master/def/de/menu_linux.cson#L3)

Ref. [no3 in CONTRIBUTING.md](https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md#translation-guide)

**Comment by liuderchi** [comment]
NOTE: todo for myself

- [ ] add @an7oine to CODEOWNER, contributor list in README.md

**Comment by an7oine** [comment]
Personally I find the bracketed hotkey letters rather irritating. Why do we need those? Such hotkeys are not present in vanilla Atom w/o this package installed. 

Perhaps you could add a package option to disable them.

nayttokuva 2018-3-1 kello 22 34 15

**Comment by liuderchi** [comment]
> Such hotkeys are not present in vanilla Atom w/o this package installed.

@an7oine Actually those hotkeys works in vanilla Atom (in a non-irritating way).

Let me explain it: 

1. These are [atom source code for menu](https://github.com/atom/atom/blob/master/menus/win32.cson#L3) for windows/linux platform: 
```coffeescript
'menu': [
  {
    label: '&File'
    submenu: [
# ...
```

For linux/windows users they could press <kbd>Alt</kbd>+<kbd>F</kbd> to navigate to `File` menu. 

2. For Translated value containing letter `F`, you can edit cson it in similar way.
e.g. [`value: "&Fichier"` (fr/menu_linux.cson)](https://github.com/liuderchi/atom-i18n/blob/master/def/fr/menu_linux.cson#L3)

3. Unfortunately, for Translated value without letter `F` (like CJK), the only way to preserve it is _wrapped in bracket_.
e.g. [`value: "Datei(&F)"` (de/menu_linux.cson)](https://github.com/liuderchi/atom-i18n/blob/master/def/de/menu_linux.cson#L3)


---

I tried to express those in [Translation Guide no.3](https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md#translation-guide).
Please let me know where the wording can be improved. 🙏

**Comment by liuderchi** [comment]
@an7oine Thank you for your contribution 💯 
Will merge this ASAP

**Pull request merged**